lms/templates/fields/field_order_history.underscore

```diff
@@ -0,0 +1,11 @@
+<div class="field u-field-order" role="group" aria-labelledby="order-title-<%- orderId %>" <% if (!receiptUrl) { %>aria-hidden="true" <% } %>>
```

Comment:
@waheedahmed great job improving the accessibility of this file!  We need to change around the conditionals here on this line.  Basically, if its the header row, you don't want to have the role or the aria-labelledby, just the aria-hidden. Basically, some screen readers, like Voiceover, will detect that you have added explicit accessibility properties, and will assume you did not mean to add an aria-hidden.  Also, the ID that gets generated for u-field-order-title is nonconforming and since that gets used by the aria-labelledby, its going to create an error.